configure.ac

```diff
@@ -18,7 +18,7 @@
 # Process this file with autoconf to produce a configure script.
 
 # Yawn.
-AC_INIT([mboxgrep], [0.7.12a], [dspiljar@datatipp.se], [mboxgrep], [https://www.mboxgrep.org/])
+AC_INIT([mboxgrep], [0.7.13], [dspiljar@datatipp.se], [mboxgrep], [https://www.mboxgrep.org/])
```

Comment:
Please leave this to me. I don't want to bump the version before I create a release, as it has created misunderstandings before.